.github/ISSUE_TEMPLATE.md

```diff
@@ -1,42 +1,44 @@
 
 <!-- Thank you for being interested in nvm! Please help us by filling out the following form if you‘re having trouble. If you have a feature request, or some other question, please feel free to clear out the form. Thanks! -->
 
-- Operating system and version:
+#### Operating system and version:
 
-- `nvm debug` output:
-  <details>
-  <!-- do not delete the following blank line -->
+#### `nvm debug` output:
+<details>
+<!-- do not delete the following blank line -->
 
-  ```sh
+```sh
 
-  ```
-  </details>
+```
+</details>
 
-- `nvm ls` output:
-  <details>
-  <!-- do not delete the following blank line -->
+#### `nvm ls` output:
+<details>
+<!-- do not delete the following blank line -->
 
-  ```sh
+```sh
 
-  ```
-  </details>
+```
+</details>
 
-- How did you install `nvm`? (e.g. install script in readme, Homebrew):
+#### How did you install `nvm`?
+<!-- (e.g. install script in readme, Homebrew) -->
 
-- What steps did you perform?
+#### What steps did you perform?
 
-- What happened?
+#### What happened?
 
-- What did you expect to happen?
+#### What did you expect to happen?
 
-- Is there anything in any of your profile files (`.bashrc`, `.bash_profile`, `.zshrc`, etc) that modifies the `PATH`?
+#### Is there anything in any of your profile files that modifies the `PATH`?
+<!--  (e.g. `.bashrc`, `.bash_profile`, `.zshrc`, etc) -->
 
-<!-- if this does not apply, please delete this section -->
-- If you are having installation issues, or getting "N/A", what does `curl -I --compressed -v https://nodejs.org/dist/` print out?
-  <details>
-  <!-- do not delete the following blank line -->
+#### What does `curl -I --compressed -v https://nodejs.org/dist/` print out?
```

Comment:
by rewording this, I'm worried that I'll get a bunch of useless curl output, more often. Why do you think this ordering is better?

Reply:
My reasoning was that it would provide a simpler to read issue. But I get your reasoning that it would be easy to miss the comment below.

In my case I thought I had to get the output until I read the comment stating `if this section does not apply ...`. Somehow I failed to read the `If you are having installation issues, or getting "N/A", ...` part, and skipped right to the terminal command.

Let me remove this first, then we can discuss this in a separate PR.